recipe/meta.yaml

```diff
@@ -11,11 +11,11 @@ build:
 requirements:
   run:
     - pangeo-dask =2020.04.04
-    - dask-labextension =1.1*
-    - jupyter-server-proxy =1.3*
-    - jupyterhub =1.1*
+    - dask-labextension =2.0.1
+    - jupyter-server-proxy =1.3.2
+    - jupyterhub =1.1.0
     - jupyterlab =1.2*
```

Comment:
The bot is having trouble parsing the jupyterlab version scheme. We can manually bump this to 2.1.0 if we want.

Reply:
if we bump to jlab 2, all lab extensions need updating. looks like dask-labextension has been and i imagine most other widely used ones have been by now, so i'm okay with it